tests/lib/rules/avoid-leaking-state-in-ember-objects.js

```diff
@@ -0,0 +1,121 @@
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/avoid-leaking-state-in-ember-objects');
+const RuleTester = require('eslint').RuleTester;
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+const eslintTester = new RuleTester();
+eslintTester.run('avoid-leaking-state-in-ember-objects', rule, {
+  valid: [
+    {
+      code: 'export default Foo.extend();',
+      parserOptions: { ecmaVersion: 6, sourceType: 'module' },
```

Comment:
You can pass shared options to the `new RuleTester()` line to avoid duplicating them for each test case.